velox/functions/prestosql/StringFunctions.h

```diff
@@ -154,6 +154,36 @@ struct HmacSha512Function {
   }
 };
 
+/// spooky_hash_v2_32(varbinary) -> varbinary
+template <typename T>
+struct SpookyHashV2_32Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  template <typename TTo, typename TFrom>
```

Comment:
Why are you using template variables TTo and TFrom when the input and output type of this function can only be Varbinary ? The rest of the functions in this file are using the specific types only.